README.md

```diff
@@ -70,13 +71,19 @@ Make sure to write tests for any optionals you implement!
 - Use a CSV file for loading data
 - Create your own tests to verify method correctness.
 
+### Testing
+You enter Wave 2 with all tests from Wave 1 passing. In Wave 2, you will take the test stubs that have already been provided for you in the `order_spec.rb` file and add the code which will get these tests to pass. You should run the tests regularly alongside the code you are writing in the `Order` class itself.
```

Comment:
Instead of writing code to get these tests to pass, that should be, "In Wave 2, you will **fill in** the test stubs that have already been provided for you in the `order_spec.rb` file and add the code which will get these tests to pass."